skfp/preprocessing/filters/tice_herbicides.py

```diff
@@ -0,0 +1,96 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumHBA, CalcNumHBD, CalcNumRotatableBonds
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class TiceHerbicidesFilter(BaseFilter):
+    r"""
+    Tice rule for herbicides.
+
+    Rule established based on statistical analysis of herbicide molecules [1]_.
```

Comment:
I would also explicitly add `Designed specifically for herbicides, not general pesticides or other agrochemicals.`

Reply:
Similarly for insecticide filter